src/styl/imports/dark.styl

```diff
@@ -0,0 +1,37 @@
+.container.dark
+	background-color #222
+	color white
+	
+	.tooltip
+		color #222
+		background-color white
+		border 5px solid white
+	
+	section.main 
```

Comment:
Stylus allows you to group all these settings like that:

```styl
section.main
  .heading
    h1, p
      color white
  .content
    background-color #222 
    .item
      .title
...
```

In this case I think you can group everything like this:
```styl
.container.dark
  section.main
    h1, p
      color white
    a
      color #d11717
    .content
      background-color #222
```

Reply:
But the second snippet as it is, won't change the color of the text. If I am not that precise, the colors will remain unaffected. Should I still group them like this?

Reply:
My bad I missed some `&`

Here's a working version
```stylus
body.dark
	background-color #222 !important

	& #color-switch
		color white !important

	& h1, p
		color white !important
	& a
		color #d11717 !important
```

Not sure is the important are needed